app/assets/stylesheets/search.scss

```diff
@@ -59,7 +60,50 @@
   border-radius: 0.5em !important;
   color: $my-black;
   padding-left: 0.7em !important;
+}
+.page-number {
+  display: inline;
+  ul li {
+    display: inline;
+    color: $my-black;
+  }
+
+  a {
+    color: $my-black;
```

Comment:
Properties should be ordered border, color, display, padding-left, padding-right, text-decoration